File: backend/app/schemas/availability.py
# app/schemas/availability.py
from pydantic import BaseModel
from typing import List
from datetime import date # For date_checked

class AvailabilityResponse(BaseModel):
    available_slots: List[str] # List of "HH:MM" strings
    date_checked: date         # YYYY-MM-DD, the date for which availability was checked
    timezone_queried: str      # The tenant's timezone string (e.g., "America/New_York")
    # Optional: could include total duration of services checked for context
    # services_duration_minutes: int 

File: frontend/src/types/Availability.ts
// src/types/Availability.ts
export interface AvailabilityResponse {
    available_slots: string[]; // Array of "HH:MM"
    date_checked: string;      // "YYYY-MM-DD"
    timezone_queried: string;  // e.g., "America/New_York"
}
